package.json

```diff
@@ -93,7 +93,8 @@
   },
   "resolutions": {
     "graphql": "15.4.0",
-    "@jahia/data-helper/graphql": "^15.4.0"
+    "@jahia/data-helper/graphql": "^15.4.0",
+    "three": "^0.125.0"
```

Comment:
Dependency is coming from `react-file-viewer`  but even latest version doesn't incorporate the fixed dependency `"three": "^0.125.0"`. Overriding with resolutions for now.

Issue opened here: https://github.com/plangrid/react-file-viewer/issues/186